app/Http/Controllers/DocsController.php

```diff
@@ -24,25 +24,24 @@ class DocsController extends Controller
      */
     public function __invoke(Documentation $docs, string $page = null)
     {
+        $defaultVersion = config('site.defaultVersion');
+
         if ($page === null) {
             return redirect()->route('docs', [self::DEFAULT_PAGE]);
         }
 
-        if (! $docs->exists(config('site.defaultVersion'), $page) || in_array($page, self::EXCLUDED)) {
+        if ($docs->doesntExists($defaultVersion, $page) || in_array($page, self::EXCLUDED)) {
```

Comment:
I think there is no need to use a new function for that

Reply:
Yeah agree, reverted now